.gitlab-ci.yml

```diff
@@ -41,7 +41,7 @@ pull-request-validation:
     - wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
     - echo "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-11 main" | tee -a /etc/apt/sources.list
     - apt update
-    - apt install -y clang-format-11
+    - DEBIAN_FRONTEND=noninteractive apt install -y clang-format-11
```

Comment:
should not be part of this pr

Reply:
I cherry-picked this to pass the CI, this PR will be automatically removed after the final rebase against #3715